pkg/apis/kops/networking.go

```diff
@@ -128,6 +128,54 @@ type AmazonVPCNetworkingSpec struct {
 
 // CiliumNetworkingSpec declares that we want Cilium networking
 type CiliumNetworkingSpec struct {
-	Debug  bool   `json:"debug,omitempty"`
-	Tunnel string `json:"tunnel,omitempty"`
+	Version string `json:"tunnel,omitempty"`
+
+	AccessLog                string            `json:"access-log,omitempty"`
```

Comment:
I wonder is this would be a candidate to using https://github.com/gambol99/kops/blob/node_registration/pkg/flagbuilder/build_flags.go#L44 as a template function rather than having to fiddle with if statements everywhere ..

Reply:
@gambol99 what do you mean by "fiddling with if statements everywhere"? Do you mean adding all these config options as cli options for `kops create cluster`?